CHANGELOG.md

```diff
@@ -25,6 +25,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.27.6] - 2024-02-09
+
+### Changed
+- Changed the WARNING messages about the deleted BOS sessions to DEBUG messages for `sat status`
+- Change the "Most Recent Image" column to show the IMS image ID instead of image name for `sat status`
```

Comment:
Minor comment, but I suggest wrapping lines at 80 chars to match existing convention in file. Also suggest "Change" -> "Changed", so they match.

```suggestion
- Changed the WARNING messages about the deleted BOS sessions to DEBUG messages
  for `sat status`
- Changed the "Most Recent Image" column to show the IMS image ID instead of
  image name for `sat status`
```